tests/sagemaker/mock/__init__.py

```diff
@@ -1076,6 +1077,6 @@ def response_object(self):
 
 
 # Create a SageMaker backend for EC2 region: "us-west-2"
-sagemaker_backends = {"us-west-2": SageMakerBackend()}
+sagemaker_backends = BackendDict(SageMakerBackend, "sagemaker")
```

Comment:
why change the key from "us-west-2" to "sagemaker" ?

Reply:
See https://github.com/spulec/moto/blob/7d76ee050ab98be790d4d9f1f8d91e3c3a23db38/moto/core/utils.py#L415

```python
class BackendDict(dict):
    """
    Data Structure to store everything related to a specific service.
    Format:
      [account_id: str]: AccountSpecificBackend
      [account_id: str][region: str] = BaseBackend
    Full multi-account support is not yet available. We will always return account_id 123456789012, regardless of the input.
    To not break existing usage patterns, the following data access pattern is also supported:
      [region: str] = BaseBackend
    This will automatically resolve to:
      [default_account_id][region: str] = BaseBackend
    """

    def __init__(
        self, backend, service_name, use_boto3_regions=True, additional_regions=None
    ):
        self.backend = backend
        self.service_name = service_name
        self._use_boto3_regions = use_boto3_regions
        self._additional_regions = additional_regions
```

Reply:
The second argument is `service_name`. It looks like we don't need to specify a region.